app/helpers/schema_helper.py

```diff
@@ -145,13 +134,12 @@ def get_aliases(cls, survey_json):
         for block in cls.get_blocks(survey_json):
             for question in cls.get_questions_for_block(block):
                 for answer in question['answers']:
-                    if 'alias' in answer and answer['alias'] not in aliases:
+                    if 'alias' in answer:
+                        assert answer['alias'] not in aliases, 'Duplicate alias found: ' + answer['alias']
```

Comment:
I'd question the need for an assert here at all. It's not a bad thing per se, and you may have a good reason for having it here, I don't know. But I think having a test in `test_schema_validation.py` and catching it at build time is sufficient. Since we're not dealing with the AssertionError at runtime anyway. Your call though.